### DISCUSSION TITLE
vscode、edge一些软件无法切换中文输入法怎么解决？

### DISCUSSION THREAD

**Discussion by ydc97** [prasanthrangan/hyprdots#1789] [Q&A]
vscode、edge一些软件无法切换中文输入法怎么解决？

**Comment by thyeun** [comment]
@ydc97 在你的```code-flags.conf```加入```--enable-wayland-ime```, 我沒用```edge```,你應該也可以在```edge-flags.conf```加入。

![240901_11h43m02s_screenshot](https://github.com/user-attachments/assets/42ddd114-2694-4ecd-832b-828f9d4540ae)

**Reply by ydc97** [reply]
好的谢谢我试试
